src/com/itacademy/eshop/shop/Eshop.java

```diff
@@ -15,6 +15,10 @@ public Eshop(String name, ArrayList<Product> products) {
         this.products = products;
     }
 
+    public void setName(String _name) {
```

Comment:
Generally, the use of underscores in variable naming is against the code conventions of Java. But it won't break anything 😄